tests/lookup/tests.py

```diff
@@ -97,7 +97,11 @@ def test_iterator(self):
 
     def test_count(self):
         # count() returns the number of objects matching search criteria.
-        self.assertEqual(Article.objects.count(), 7)
+        with self.assertNumQueries(1) as ctx:
+            self.assertEqual(Article.objects.count(), 7)
+        sql = ctx.captured_queries[0]['sql']
+        self.assertTrue(sql.startswith('SELECT COUNT(*) '), sql)
```

Comment:
Done, also moved the test to the aggregate tests where it seemed a bit more suitable